core/src/com/mygdx/minigolf/model/components/Player.java

```diff
@@ -1,4 +1,25 @@
 package com.mygdx.minigolf.model.components;
+import com.badlogic.ashley.core.Component;
+
+
+public class Player implements Component{
+    private int strokes = 0;
+    private boolean completed = false;
+
+    public boolean isCompleted() {
+        return completed;
+    }
+
+    public void setCompleted(boolean completed) {
+        this.completed = completed;
+    }
```

Comment:
Blir vi noen gang å sette completed = false? Hvis denne kun brukes når en spiller når et hull så kan den vel endres til
```
void setCompleted() { this.completed = true; }
```

Reply:
Kanskje bruk et annet navn enn setCompleted da, siden det vanligvis brukes for setere